### DISCUSSION TITLE
using Jbrowse with vue3

### DISCUSSION THREAD

**Discussion by walidsebai** [GMOD/jbrowse-components#4171] [Q&A]
Hello,
first of, thanks for the great and open product.
My team and I have been trying to build a custom genome browser with vue3 and hoped to use Jbrowse2, but the components are made and exported with react only, has anyone figured a clean way to integrate the component in another framework than react?

**Comment by cmdcolin** [comment]
thanks for the kind words :)

i think you will need a type of react to vue bridge. you might want to check out the work @emiliorighi in this repo https://github.com/emiliorighi/jbrows2vue-demo and also this thread https://github.com/GMOD/jbrowse-components/discussions/2930

**Reply by emiliorighi** [reply]
Hi, you can also take a look at https://github.com/guigolab/biogenome-portal where I have integrated JBrowse2 into a Vue3 app.
Let me know if you need some help
